src/components/dialogs/name-check-issues.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-dialog v-model="display" width="45rem" persistent :attach="attach">
+    <v-card>
+      <v-row no-gutters>
+        <v-col cols="6" class="text-end">
+          <v-icon class="mr-n4" :color="options.iconColor" size="2.5rem">{{ options.icon }}</v-icon>
+        </v-col>
+        <v-col>
+          <v-btn cols="6" class="close-btn outlined" icon :ripple="false" size="2.5rem" @click="proceed(false)">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+        </v-col>
+      </v-row>
+      <p class="dialog-title ma-0 pt-5">
+        {{ options.title }}
+      </p>
+      <v-card-text class="pt-10">
```

Comment:
Seems a bit weird to have
v-card
-- v-row
-- p
-- etc

instead of
v-card
-- v-card-text
---- v-row
---- p
---- etc

Reply:
sure I can try changing that. Not sure if that'll mean cascading changes with v-card-text styling though. I might just take that out entirely if needed

Reply:
the reason it's like that is to separate the icons at the top from the text grid so that it displays correctly when the screen changes size

Reply:
It's just an expectation of not mixing row/col and card blocks.

Reply:
Maybe v-card has other blocks you can use?

https://vuetifyjs.com/en/components/cards/#usage

Reply:
Ah okay yes I will update it so I'm not mixing them

Reply:
Looks good!